### ISSUE TITLE
you can't tab though detects after grid update

### ISSUE THREAD

**Issue by patrickkettner** [Modernizr/modernizr-neue#50]
cc @joecritch

**Comment by joecritch** [comment]
@patrickkettner hey, do you literally mean the "tab" key? or arrows? I never noticed this as a feature actually, but I can compare the old to new and take a look.

**Comment by patrickkettner** [comment]
Yes, before you could just hold down the tab key and keep on keeping on

**Comment by patrickkettner** [comment]
this happens in chrome, not in edge or firefox
